app/access/models.py

```diff
@@ -29,3 +47,7 @@ def save(
         """Validates the model before writing it to the database."""
         self.full_clean()
         super().save(force_insert, force_update, using, update_fields)
+
+    def disable(self) -> None:
+        self.deleted_at = timezone.now()
```

Comment:
Maybe add a note on why datetime.now() is not a good idea here. Not obvious as we saw when the tests failed.